scripts/node/README.md

```diff
@@ -0,0 +1,42 @@
+# Steps to benchmark node scanning 
+
+## Pre steps:
+
+quay.io/repository/rhacs-eng/sandbox:benchmark-node-analyze-1.0 is an Ubi8 based image with vim, RPM and RPM-build installed. Those packages are necessary for benchmark node analyze function
+
+back-up image: `quay.io/rh_ee_yli3/nodes:benchmark-node-analyze-1.0`
```

Comment:
What is the purpose of the backup image?

Should we move it from the `yli3` namespace into something generic?